src/lib/components/list/QItemSection.svelte

```diff
@@ -15,14 +15,17 @@
       $$slots.headline && [$$slots.line1, $$slots.line2, $$slots.line3].filter(Boolean).length >= 2;
   }
 
-  $: classes = createClasses([`q-item__section `, `q-item__section--${type}`, userClasses]);
+  $: classes = createClasses([type], {
+    component: "q-item__section",
+    userClasses,
+  });
 </script>
 
 <div class={classes} {...$$restProps}>
   {#if type === "content"}
-    {#if [$$slots.headline, $$slots.line2, $$slots.line2, $$slots.line3].some(Boolean) === false}
+    <!-- {#if [$$slots.headline, $$slots.line2, $$slots.line2, $$slots.line3].some(Boolean) === false}
```

Comment:
Why is this commented out? Can we remove it?

Reply:
I was not sure why it was here so I commented out in case I had to go back to it. I was right because I had to fix it in the last commits